Doc/howto/clinic.rst

```diff
@@ -1471,7 +1471,7 @@ preset configurations, as follows:
     The default filename is ``"{dirname}/clinic/{basename}.h"``.
 
   ``buffer``
-    Save up all most of the output from Clinic, to be written into
+    Save up almost all of the output from Clinic, to be written into
```

Comment:
“Save up most of . . .” would be simpler I think, assuming that’s what @larryhastings meant. But that leaves the question of what is and isn’t saved. Next paragraph mentions suppressing some stuff, but only writing stuff to _block_ and _file_, which doesn’t help me. Perhaps _file_ should be _buffer_?